### PR TITLE
Init DevTools Proto

### PR DESCRIPTION
**Pull request by SHeimlich**
- Create DevTools Init Proto
- Send a DevTools Init Proto when a websocket is connected.

cc: @piotrswigon @jasonwyatt

### CODE REVIEW

File: java/arcs/android/devtools/devtools_message.proto [hunk 1]
```diff
@@ -16,3 +17,12 @@ message DevToolsProxyMessageProto {
   string storage_key = 2;
   StoreType store_type = 3;
 }
+
+message DevToolsInitProto {
+  int32 dev_tools_version = 1;
+  enum Platform {
+    ANDROID = 0;
```
Comment (csilvestrini) [inline java/arcs/android/devtools/devtools_message.proto:24]: Best practices are to make 0 a PLATFORM_UNKNOWN value, and start your real values from 1.

File: java/arcs/android/devtools/devtools_message.proto [hunk 2]
```diff
@@ -2,6 +2,7 @@ syntax = "proto3";
 
 package arcs.android.devtools;
 
+import "google/protobuf/wrappers.proto";
```
Comment (csilvestrini) [inline java/arcs/android/devtools/devtools_message.proto:5]: Still don't need this 😉

File: java/arcs/android/devtools/BUILD [hunk 3]
```diff
@@ -51,6 +51,7 @@ proto_library(
     srcs = ["devtools_message.proto"],
     deps = [
         "//java/arcs/android/storage:proxy_message_proto",
+        "//third_party/java/arcs/deps:protobuf_wrappers_proto",
```
Comment (csilvestrini) [inline java/arcs/android/devtools/BUILD:54]: Still don't need this 😉

**Review by csilvestrini — APPROVED** [review]

File: java/arcs/android/devtools/BUILD (hunk 3 above)
Comment (SHeimlich) [reply java/arcs/android/devtools/BUILD]: 🤦

**Pull request closed without merging**